### ISSUE TITLE
Clarification on workflow for human-in-the-loop training on Jupyter (CPSAM)

### ISSUE THREAD

**Issue by jmkotah** [MouseLand/cellpose#1206]
Hello Cellpose team!

Thank you very much for this very useful and valuable tool. I am running Cellpose 4 with CPSAM, trying to fine-tune the model for my specific images using either the GUI or on a Jupyter notebook, and had just some clarifications, in part based on the comments here: https://github.com/MouseLand/cellpose/issues/723

I've been able to try out the workflow on GUI, and from my understanding, the workflow is:
1. Run CPSAM to get masks
2. Fix masks manually by removing bad masks or adding missed masks
3. On GUI, CTRL+T to train a new model, which is iteratively done as many times as there are images
4. Ensure that all subsequent trainings are always done on a folder that also contains previously trained images seen by the model to avoid 'drift'

I am trying to implement this on a Jupyter notebook, in part because I do not currently have access to a GPU and was hoping to train the models faster using Colab. I was able to try out the code in the Jupyter notebook you provided, which works great. However, I am currently unsure about the best way to fine-tune the CPSAM model on Jupyter, especially in contrast to the way it is done in the GUI. My questions are:
 
1. What is the best way to approximate the iterative (i.e., per image) model training + manual mask annotation step done in the GUI? I can imagine locally running CPSAM on, e.g., 5 images, fixing their masks, and then uploading them all to one folder for which the base model will be re-trained on, but I am not sure if that is equivalent to creating successively training one model per image in the folder, as is done in the GUI. 
2. If that's not the same, should I then be running the train_seg function in a folder with one image, applying the model to a new image, fix the predicted masks, and upload to a new folder that now contains two images and two masks and retrain the previous model, and so on?
3. Lastly, once all training is done, what model should I use to segment new images not part of the training set? Just the last model from the last round of training? I am asking because from https://github.com/MouseLand/cellpose/issues/723, I understood that one of the options after all the training steps are done is to re-train the 'base' CPSAM model in a folder with images and masks generated throughout the previously done training. Would this be preferable to use?

Hope to hear your thoughts on this. Thank you so much for your time!

**Comment by mrariden** [comment]
We'll make a new issue for that.
